packages/@ourworldindata/grapher/src/popover/Popover.scss

```
@@ -0,0 +1,30 @@
+.popover {
+    // Backdrop - transparent mouse-event catcher while popup is visible
+    .popover__backdrop {
+        position: absolute;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        z-index: $zindex-controls-backdrop;
+    }
+
+    // Wrapper - the actual popover container
+    .popover__wrapper {
+        position: absolute;
+        background: white;
+        border-radius: 4px;
+        box-shadow: 0px 4px 23px 4px #0000000f;
+        z-index: $zindex-controls-popup;
+        overflow-y: scroll;
```

Can we instead make this?
```suggestion
        overflow-y: auto;
```

That should fix the scrollbar issue I talked about above.